### PR TITLE
fix(all): add keywords, reduce size, export modules

### PR DESCRIPTION
**Pull request by younkue**
## Issue

All #130 
JustifiedLayout.js #122 
package.json #136 
index.js, index.umd.js #132

### CODE REVIEW

**Comment by jongmoon** [comment]
@younkue LGTM, I like your changes.

**Comment by mixed** [comment]
👍 LGTM

File: src/ItemManager.js [hunk 1]
```diff
@@ -74,15 +74,9 @@ export default class ItemManager {
...
-		if (typeof start !== "undefined") {
-			if (typeof end !== "undefined") {
-				return ItemManager.pluck(this._data.slice(start, end + 1), property);
-			} else {
-				return ItemManager.pluck(this._data.slice(start, start + 1), property);
-			}
-		} else {
-			return ItemManager.pluck(this._data, property);
-		}
+		const data = typeof start === "undefined" ? this._data : this._data.slice(start, (typeof end === "undefined" ? start : end) + 1);
```
Comment (mixed) [inline src/ItemManager.js]: I think that It repeat same code that undefined check if condition.

File: src/ItemManager.js [hunk 2]
```diff
@@ -176,15 +170,10 @@ export default class ItemManager {
...
 	}
 	get(start, end) {
-		if (typeof start !== "undefined") {
-			if (typeof end !== "undefined") {
-				return this._data.slice(start, end + 1);
-			} else {
-				return this._data.slice(start, start + 1);
-			}
-		} else {
+		if (typeof start === "undefined") {
```
Comment (mixed) [inline src/ItemManager.js]: undefined check if condition.

File: src/ItemManager.js [hunk 3]
```diff
@@ -176,15 +170,10 @@ export default class ItemManager {
...
 			return this._data.concat();
 		}
+		return this._data.slice(start, (typeof end === "undefined" ? start : end) + 1);
```
Comment (mixed) [inline src/ItemManager.js]: undefined check if condition.

**Pull request merged**